### PR TITLE
Support oneOfs & anyOfs in operation request & response schema

### PR DESCRIPTION
**Pull request by lgarber-akamai**
## 📝 Description

This pull request adds support for parsing [OpenAPI oneOfs and anyOfs](https://swagger.io/docs/specification/v3_0/data-models/oneof-anyof-allof-not/) in both the request and response schema for operations. 

Additionally, this PR adds support for rendering each root-level request oneOf separate section in the generated command help pages.

## ✔️ How to Test

The following test steps assume you have pulled down this pull request locally and run the following command:

```shell
make SPEC='https://gist.githubusercontent.com/lgarber-akamai/07485a643e8329a5c16bec775e8066c4/raw/56f4491ba6e239c9cad736f05ff2d1ab0608098c/openapi.json' install
```

**NOTE: The overridden specification is necessary because the currently published OpenAPI specification does not yet make use of oneOfs.**

### Unit Testing

```
make testunit
```

### Integration Testing

```
make testint
```

### Manual Testing

1. Run the following command the view the help page of the `nodebalancers config-create` command:

```shell
linode-cli nodebalancers config-create --help
```

2. Ensure the output contains separate categories for each configuration protocol.
3. Create a NodeBalancer using the following command:

```shell
linode-cli nodebalancers create \
  --region us-mia \
  --label my-nodebalancer
``` 

4. Create a NodeBalancer configuration using the following command, replacing the `MYIDHERE` with the ID of the newly created NodeBalancer:

```shell
linode-cli nodebalancers config-create MYIDHERE \
  --port 440 \
  --protocol http \
  --algorithm roundrobin \
  --stickiness http_cookie \
  --check http_body \
  --check_interval 90 \
  --check_timeout 10 \
  --check_attempts 3 \
  --check_path "/test" \
  --check_body "it works"
```

5. Ensure the NodeBalancer configuration has been created successfully.

### Improved Help Page

Improved Help Page

### CODE REVIEW

File: linodecli/baked/request.py [hunk 1]
```diff
@@ -134,66 +137,70 @@ def _parse_request_model(schema, prefix=None, parent=None, depth=0):
...
+            args.append(
+                OpenAPIRequestArg(
+                    k,
+                    v,
+                    required,
+                    prefix=prefix,
+                    parent=parent,
+                    depth=depth,
                 )
+            )
```
Comment (lgarber-akamai) [inline linodecli/baked/request.py:204]: This logic is all the same, I just replaced the if statement with a guard clause

File: linodecli/baked/request.py [hunk 2]
```diff
@@ -134,66 +137,68 @@ def _parse_request_model(schema, prefix=None, parent=None, depth=0):
...
+    properties, required = _aggregate_schema_properties(schema)
+
+    if properties is None:
+        return args
+
+    for k, v in properties.items():
+        if (
+            v.type == "object"
+            and not v.readOnly
+            and len(_aggregate_schema_properties(v)[0]) > 0
```
Comment (lgarber-akamai) [inline linodecli/baked/request.py:149]: This is necessary to account for nested anyOfs and oneOfs.

**Review by jriddle-linode — APPROVED** [review]
Working locally, looks good to me!

**Review by zliang-akamai — APPROVED** [review]
Tested and looks good to me!

**Pull request merged**